### PR TITLE
ILI-23: Added possibility to select where to send the lab order

### PR DESCRIPTION
**Pull request by oskarhinc**

### CODE REVIEW

**Comment by areklalo** [comment]
I believe you can do this using the concept_answer table.

File: omod/src/main/webapp/resources/htmlforms/Lab.xml [hunk 1]
```diff
@@ -3025,12 +3025,22 @@
 	<br/>
 	<br/>
 	<div id="datePr">
-	Date de Prochaine Visite :<obs conceptId="CIEL:5096" allowFutureDates="true"/><br/><br/>
-	Nom d'auteur de la fiche :<encounterProviderAndRole encounterRole="322fc5ba-2ed6-44cf-8ebc-963dc40322af" autocompleteProvider="true" default="currentUser"/> <br/><br/>
+		Date de Prochaine Visite :<obs conceptId="CIEL:5096" allowFutureDates="true"/><br/><br/>
+		Nom d'auteur de la fiche :<encounterProviderAndRole encounterRole="322fc5ba-2ed6-44cf-8ebc-963dc40322af" autocompleteProvider="true" default="currentUser"/> <br/>
+
+		Envoyez le résultat de laboratoire à: <br/>
+		<input id="send-lab-order-to-SCC" type="checkbox" />
```
Comment (dserkowski) [inline omod/src/main/webapp/resources/htmlforms/Lab.xml]: It is required to add the name attribute in order to make it possible to fetch this data on the backend.

File: omod/src/main/webapp/resources/htmlforms/Lab.xml [hunk 2]
```diff
@@ -3025,12 +3025,22 @@
...
+		<label for="send-lab-order-to-SCC">SCC via Sedish</label>
+		<includeIf velocityTest="$fn.globalProperty('labintegration.openElis.url') != ''">
+			<input id="send-lab-order-to-OpenELIS" type="checkbox" />
```
Comment (dserkowski) [inline omod/src/main/webapp/resources/htmlforms/Lab.xml]: As above

File: omod/src/main/webapp/resources/htmlforms/Lab.xml (hunk 1 above)
Comment (dserkowski) [reply omod/src/main/webapp/resources/htmlforms/Lab.xml]: outdated comment. Now, we pass order destination by a a generic text concept

**Pull request merged**